tests/core/MongoDbOperationStore.spec.ts

```diff
@@ -36,8 +38,23 @@ async function constructAnchoredCreateOperation (
   transactionNumber: number,
   transactionTime: number,
   operationIndex: number): Promise<AnchoredOperation> {
-  const didDocumentTemplate = require('../json/didDocumentTemplate.json');
-  const operationBuffer = await OperationGenerator.generateCreateOperationBuffer(didDocumentTemplate, publicKey, privateKey);
+  const signingKeys = await Cryptography.generateKeyPairHex('#key2', KeyUsage.signing);
+  const service = [
+    {
+      'id': 'IdentityHub',
+      'type': 'IdentityHub',
+      'serviceEndpoint': {
+        '@context': 'schema.identity.foundation/hub',
+        '@type': 'UserServiceEndpoint',
+        'instance': [
+          'did:sidetree:value0'
+        ]
+      }
+    }
+  ];
```

Comment:
Can we create a method `OperationGenerator.createIdentityHubServiceEndpoint(instances: string[])`?

Then:

```javascript
const identityHubServiceEndpoint =  OperationGenerator.createIdentityHubServiceEndpoint(['did:sidetree:value0']);
const services = [identityServiceEndpoint];
```